src/Sculpin/Bundle/SculpinBundle/HttpKernel/KernelFactory.php

```diff
@@ -30,7 +30,11 @@ class KernelFactory
     public static function create(InputInterface $input)
     {
         $env = $input->getParameterOption(array('--env', '-e'), getenv('SCULPIN_DEBUG') ?: 'dev');
-        $debug = $env !== 'prod' && getenv('SCULPIN_DEBUG') !== '0' && !$input->hasParameterOption(array('--no-debug', ''));
+        $debug = $env !== (
+            'prod'
```

Comment:
It looks like the logic is changed on this line, since 'prod' on its own would always evaluate to true.